src/components/common/TmBalance.vue

```diff
@@ -233,6 +269,15 @@ export default {
       } else {
         return [this.stakingDenom]
       }
+    },
+    isMultiDenomReward() {
+      if (this.overview.rewards && this.overview.rewards.length > 0) {
+        return this.overview.rewards[0].denom !== this.overview.rewards[1].denom
```

Comment:
are multiple rewards with the same denom possible?

Reply:
yes, because of rewards from different validators.

But I don't like this logic either. This is why I want this PR luniehq/lunie-api#343 merged

Reply:
why does the overview query receive all rewards? doesn't only need a sum per denom?